cdk.json

```diff
@@ -31,36 +31,32 @@
       "aws-cn"
     ],
     "dev": {
-      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/myapp:release-1.0",
+      "IMAGE_PATH_AND_TAG": "registry:2",
       "AWS_DEFAULT_REGION": "us-east-1",
-      "PORT": "3838",
+      "PORT": "443",
       "CONTAINER_ENV": {
-        "name1": "value1",
-        "name2": "value2"
       },
       "TAGS": {
-        "CostCenter": "NO PROGRAM / 000000",
-        "OwnerEmail": "joe@sagebase.org"
+        "CostCenter": "Platform Infrastructure / 990300",
+        "OwnerEmail": "synapseeng@sagebase.org"
       },
-      "STACK_NAME_PREFIX": "myapp-dev",
-      "VPC_CIDR": "172.31.0.0/24",
-      "ACM_CERT_ARN": "arn:aws:acm:us-east-1:<DEV_ACCOUNT_ID>:certificate/<DEV_CERT_ID>"
+      "STACK_NAME_PREFIX": "registry-dev",
+      "VPC_CIDR": "172.33.0.0/24",
```

Comment:
Should not we use an existing VPC (either same as Synapse stack) or the one already set by infra?

Reply:
> Should not we use an existing VPC (either same as Synapse stack or the one already set by infra)?

it probably depends on the use case.  Using a shared VPC would certainly reduce cost however it may not provide the isolation you desire.  In this case creating a new VPC and ECS just for one micro service probably does not make sense.  The way I would approach this use case is to split up the CDK objects to create a separate shared VPC and ECS for all related Synapse micro services (i.e. docker registry, jenkins, jfrong, etc..).  That would be more work though.

Reply:
Updated to `172.30.0.0/24`.